File: CalculatorKata/CalculatorKataTests/CalculatorTests.m
#import "CalculatorTests.h"
#import "Calculator.h"

@implementation CalculatorTests

- (void)setUp {
    [super setUp];
    sut = [[Calculator alloc] init];
}

- (void)testAddMethod_zeroLengthInput_shouldReturn0 {
    int expected = 0;
    int result = [sut add:@""];

    STAssertEquals(expected, result, @"Zero length input should return 0.");
}

- (void)testAddMethod_oneLengthInput_shouldReturnEquivalent {
    int expected = 3;
    int result = [sut add:@"3"];

    STAssertEquals(expected, result, @"One length input should return equivalent.");
}

- (void)testAddMethod_twoLengthInput_shouldReturnSum {
    int expected = 8;
    int result = [sut add:@"3,5"];

    STAssertEquals(expected, result, @"Two length input should return sum.");
}


@end

File: CalculatorKata/CalculatorKataTests/CalculatorTests.swift
import XCTest
import CalculatorKata

class CalculatorTests: XCTestCase {

    func test_givenSut_whenZeroInput_thenShouldReturn0() {
        var expected = 0;
        var sut = Calculator()
        var result = sut.add("")
        XCTAssertEqualObjects(expected, result)
    }

    func test_givenSut_whenOneInput_thenShouldReturnNumericEquivalent() {
        var expected = 8;
        var sut = Calculator()
        var result = sut.add("8")
        XCTAssertEqualObjects(expected, result)
    }

}
